### PR TITLE
[XAM] XamGetStagingMode Stub

### PR DESCRIPTION
**Pull request by AdrianCassar**
Enables debug information on the dashboard
image

More info here: [MITMing the Xbox 360 Dashboard for Fun and RCE](https://landaire.net/mitming-the-xbox-360-dashboard-for-rce-and-fun/)

### CODE REVIEW

File: src/xenia/kernel/xam/xam_info.cc [hunk 1]
```diff
@@ -58,6 +58,9 @@ typedef enum _MODE { KernelMode, UserMode, MaximumMode } MODE;
 dword_result_t XamFeatureEnabled_entry(dword_t unk) { return 0; }
 DECLARE_XAM_EXPORT1(XamFeatureEnabled, kNone, kStub);
 
+dword_result_t XamGetStagingMode_entry() { return 1; }
```
Comment (The-Little-Wolf) [inline src/xenia/kernel/xam/xam_info.cc]: Shouldn't it return 0 since 1 is staging/debug mode and 0 is production?

Comment (Gliniak) [reply src/xenia/kernel/xam/xam_info.cc]: I would just add it as another config variable or attach it to "debug" config option

Comment (AdrianCassar) [reply src/xenia/kernel/xam/xam_info.cc]: Added `staging_mode` config variable to General category. However, Kernel or XConfig category might be more suitable?

Comment (Gliniak) [reply src/xenia/kernel/xam/xam_info.cc]: Probably Kernel imo

**Pull request merged**